src/poliastro/core/fixed.py

```diff
@@ -60,11 +60,11 @@ def mercury_rot_elements_at_epoch(T, d):
         Right ascension and declination of north pole, and angle of the prime meridian.
 
     """
-    M1 = 174.7910857 + 4.092335 * d
-    M2 = 349.5821714 + 8.184670 * d
-    M3 = 164.3732571 + 12.277005 * d
-    M4 = 339.1643429 + 16.369340 * d
-    M5 = 153.9554286 + 20.461675 * d
+    M1 = np.deg2rad(174.7910857 + 4.092335 * d)
```

Comment:
Looks good to me too! However, I would like to suggest one thing and know your opinion about it: we are calling several times the `np.deg2rad` function, what about implementing the following at the very beginning of the file:

```python
DEG2RAD = np.deg2rad(1)
```
so we can do:

```python
M1 = number_in_degrees * DEG2RAD
```

What do you think?

Reply:
All these functions are jitted, so I don't expect a big difference. However, I might be wrong :)

Reply:
The execution times seem to be almost the same with the approach of using `numpy` multiple times being marginally faster (`~1e-2 s`). From a short analysis, this marginal increase might be because

```py
DEG2RAD = np.deg2rad(1)
```
would not be inside any jitted function and hence introduces some more execution time, although using just `DEG2RAD` inside the jitted functions is slightly faster than this PR's implementation.

As @astrojuanlu said, the differences are not that big, so I wonder what option could we choose here?

Reply:
Yeah, let us use the original `np.deg2rad` implementation. Thanks for the comments 🚀